core-bundle/src/Resources/contao/models/FilesModel.php

```diff
@@ -96,6 +96,31 @@ class FilesModel extends Model
 	 */
 	protected static $strTable = 'tl_files';
 
+	/**
+	 * Returns the full content of the file if it exists and null otherwise.
+	 */
+	public function getContent(): ?string
+	{
+		$path = $this->getAbsolutePath();
+
+		if (!file_exists($path))
+		{
+			return null;
+		}
+
+		return (string) file_get_contents($path);
```

Comment:
I do not like this. The model should not be responsible to read the file. I would prefer to have the controller do this. WDYT?

Reply:
I put it here because I was fed up by writing the same code over and over again. It deserves its own shortcut.

Reply:
I disagree. Not the model's responsibility. @contao/developers WDYT?

Reply:
:thinking:  IMO one could argue that the model is wrapper around the resource. So this way it wouldn't matter if the content comes from the database, is a resource handle that will be read or whatever (implementation detail). I had the same discussion over entities and it boiled down to whether you wanted rich entities (models) or POPOs.

Reply:
I don't really think this should be part of the model either, also because it's basically a one-line?

Reply:
But I can leave the absolute path there? That's the one I care about more because I don't want to inject the project_dir everywhere